docs/specification/README.md

```diff
@@ -51,30 +52,19 @@ These are the magic bytes:
 
 The first record in every file must be a Header (op=0x01) and the last record must be a Footer (op=0x02).
 
-MCAP files may be **"chunked"** or **"unchunked"**. Chunked and unchunked files have different constraints on the layout of record types in the file. In chunked files, messages are grouped into optionally-compressed blocks of data before being written to disk. In an unchunked file, each message is written out uncompressed. See the diagrams below for clarity (the record types shown are described in the following section):
+MCAP files may contain a variety of record types. Specific constraints on valid usage of the record types is explained in the sections below, but in general record types may be used or not depending on the feature requirements of the consumer:
 
-#### Chunked
+- If the consumer requires fast random access to message data in the file based on topic and timestamp, they will want to use Message Index records after their Chunks, and ensure the index data section includes Channel Info and Chunk Index records.
+- If the consumer requires fast summarization of file contents, they should include a Statistics record in their index data section.
+- If the consumer wants to ensure as much data is written as possible in case of crash, they should use Message records at the top level, instead of Chunks.
```

Comment:
These feel too prescriptive (maybe not the right word). The feeling I walk away with when reading some of these is "ok why". If I want to ensure as much data is written in case of a crash why should I not use chunks? Doesn't that depend on how I've implemented my writer? I can definitely implement a writer that is both crash-data-loss-resilient and uses chunks.